src/WordPress/Hooks.php

```diff
@@ -16,6 +16,8 @@ class Hooks
     protected $logger;
     protected $proxy;
 
+    const CLOUDFLARE_JSON = "CLOUDFLARE_JSON";
```

Comment:
instead of double quote, let's use single quote to match our style. I'm not sure why the linter didn't get this..